src/fragment-components/a-list.tsx

```diff
@@ -0,0 +1,272 @@
+						kind='ghost'
+						aria-label='Add Children'
+						title='Add Children'
+						onClick={(event: any) => {
+							event.stopPropagation();
+							addToList(event.currentTarget.id);
+						}}>
+						<Add16 className={actionIconStyle} />
+					</Button>
+					<Button
+						id={componentObj.id}
+						kind='ghost'
+						aria-label='Delete'
+						title='Delete'
+						onClick={(event: any) => {
+							event.stopPropagation();
+							deleteFromList(componentObj);
+						}}>
+						<TrashCan16 className={actionIconStyle} />
+					</Button>
+				</>
+			},
+			children: componentObj.items?.map((item: any) => getHierarchyListItemsFromComponentObj(item))
+		};
+	};
+
+	return <HierarchyList
+		title={title}
+		className={LayoutStyle}
+		items={getHierarchyListItemsFromComponentObj(fragment.data)?.children}
+		onListUpdated={(updatedItems: any[]) => {
+			setFragment({
+				...fragment,
+				data: getReorderedComponentObjFromHierarchyListItem({ id: 1, children: updatedItems }, fragment.data)
+			});
+		}}
+		editingStyle='single'
+	/>;
+};
+
+export const AListSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	const [isAccordionOpen, setIsAccordionOpen] = useState({} as any);
+	useEffect(() => {
+		setIsAccordionOpen({
+			small: selectedComponent.smallSpan || selectedComponent.smallOffset
+		});
+	}, [selectedComponent]);
+	return <>
+		<TextInput
+		value={selectedComponent.legendName}
+		labelText='Legend name'
+		placeholder='Legend name'
+		onChange={(event: any) => {
+			setComponent({
+				...selectedComponent,
+				legendName: event.currentTarget.value
+			});
+		}} />
+		<Accordion align='start'>
+			<AccordionItem
+			id={selectedComponent.id}
+			title='Items list'
+			className='layout-widget'
+			open={isAccordionOpen.small}>
+				<ListItemsWidget selectedComponent={selectedComponent} setComponent={setComponent}/>
+			</AccordionItem>
+		</Accordion>
+	</>;
+};
+
+export const AListCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => {
+		setComponent({
+			...selectedComponent,
+			codeContext: {
+				...selectedComponent.codeContext,
+				name: event.currentTarget.value
+			}
+		});
+	}}
+/>;
+
+export const AList = ({
+	children,
+	componentObj,
+	selected,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		selected={selected}
+		headingCss={css`width: fit-content; min-width: 9rem;`}
+		componentObj={componentObj}
+		{...rest}>
+			<legend className={cx(css`margin-left: 3px;`, 'bx--label')}>
+				{componentObj.legendName}
+			</legend>
+			<OrderedList className={OrderedListStyle}>
+				{children}
+			</OrderedList>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AList,
+	settingsUI: AListSettingsUI,
+	codeUI: AListCodeUI,
+	keywords: ['list'],
+	name: 'List',
+	type: 'list',
+	defaultComponentObj: {
+		type: 'list',
+		legendName: 'Ordered List',
```

Comment:
```suggestion
		legendName: 'Ordered list',
```

sentence capitalization...